projects/ion/src/lib/switch/switch.component.ts

```diff
@@ -19,23 +25,23 @@ import { IonSwitchProps } from './types';
     },
   ],
   host: {
-    '[attr.data-size]': 'size',
+    '[attr.data-size]': 'size()',
   },
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class IonSwitchComponent implements ControlValueAccessor {
-  @Input() key!: IonSwitchProps['key'];
-  @Input() value: IonSwitchProps['value'] = false;
-  @Input() size: IonSwitchProps['size'] = 'sm';
-  @Input() disabled: IonSwitchProps['disabled'] = false;
-  @Output() atValueChange: IonSwitchProps['atValueChange'] =
-    new EventEmitter<boolean>();
+  key = input<IonSwitchProps['key']>();
```

Comment:
```suggestion
  key = input.required<IonSwitchProps['key']>();
```
shouldn't it be a required input?